packages/preview/ntnu-thesis/0.1.0/template/main.typ

```diff
@@ -0,0 +1,641 @@
+#import "../lib.typ": *
```

Comment:
This should be `#import "@preview/ntnu-thesis:0.1.0": *` so that it actually works when people start from the template.